pkg/provider/aws/action/openshift-snc/cloud-config

```diff
@@ -16,6 +16,7 @@ runcmd:
   - chmod 0644 /opt/crc/pass_developer
   - echo "{{ .PublicIP }}" > /opt/crc/eip
   - chmod 0644 /opt/crc/eip
+  - sudo xfs_growfs -d /
```

Comment:
sorry, remove sudo here as this is already being executed as root during boot time